Parse/Internal/FieldOperation/PFFieldOperation.m

```diff
@@ -153,9 +153,8 @@ - (PFFieldOperation *)mergeWithPrevious:(PFFieldOperation *)previous {
                                               withNumber:((PFIncrementOperation *)previous).amount];
         return [PFIncrementOperation incrementWithAmount:newAmount];
     } else {
-        @throw [NSException exceptionWithName:NSInternalInconsistencyException
-                                       reason:@"Operation is invalid after previous operation."
-                                     userInfo:nil];
+        [NSException raise:NSInternalInconsistencyException format:@"Operation is invalid after previous operation."];
+        return nil;
```

Comment:
Nit: Move this outside of the `else` scope, so the method that returns something actually returns something.